admin-api-server/src/nft/nft.controller.ts

```diff
@@ -97,4 +89,14 @@ export class NftController {
       updateNft.value,
     );
   }
+
+  #validateFilterParams(params: NftPaginationParams): void {
+    this.#validatePaginationParams(params);
+  }
```

Comment:
Are you planning on adding more Validation in this function ? if not why having it as it's calling the one just below